For an unknown reason in cross it dosen't detect nss subdirectories.

--- crypto/nss_util.cc	2020-09-26 17:36:50.201101961 +1000
+++ crypto/nss_util.cc	2020-09-26 17:35:35.329796220 +1000
@@ -4,7 +4,7 @@
 
 #include "crypto/nss_util.h"
 
-#include <nss.h>
+#include <nss/nss.h>
 #include <pk11pub.h>
 #include <plarena.h>
 #include <prerror.h>
--- net/cert/x509_util_nss.cc	2020-09-27 21:11:11.393820715 +1000
+++ net/cert/x509_util_nss.cc	2020-09-27 21:11:29.340897670 +1000
@@ -7,7 +7,7 @@
 #include <cert.h>  // Must be included before certdb.h
 #include <certdb.h>
 #include <cryptohi.h>
-#include <nss.h>
+#include <nss/nss.h>
 #include <pk11pub.h>
 #include <prerror.h>
 #include <secder.h>
